0035-ilp-over-http/0035-ilp-over-http.md

```diff
@@ -52,7 +61,48 @@ Content-Type: application/octet-stream
 < Body: Binary OER-Encoded ILP Fulfill or Reject Packet >
 ```
 
-All ILP Packets MUST be returned with the HTTP status code `200: OK`.
+### ILP Fulfill/Reject
+
+#### Request
+
+```http
+POST /ilp HTTP/x.x
+Host: example.com
+Content-Type: application/octet-stream
+Authorization: Bearer zxcljvoizuu09wqqpowipoalksdflksjdgxclvkjl0s909asdf
+Idempotency-Key: 8988dd17-55e4-40e0-9c57-419d81a0e3a5
+
+< Body: Binary OER-Encoded ILP Fulfill or Reject Packet >
+```
+
+- **Path** &mdash; A connector MAY specify any HTTP path for their peer to send ILP packets to.
+- **Host Header** &mdash; The standard [HTTP Host Header](https://tools.ietf.org/html/rfc2616#section-14.23) indicating the domain of the HTTP server the Request is sent to.
+- **Content-Type Header** &mdash; MUST be set to `application/octet-stream`.
+- **Idempotency Key Header** &mdash; Same idempotency key used in the corresponding ILP Prepare.
+- **Body** &mdash; ILP Packet encoded using OER, as specified in [RFC 27: Interledger Protocol V4](./0027-interledger-protocol-4/0027-interledger-protocol-4.md).
+
+#### Response
+
+```http
+HTTP/x.x 200 OK
+```
+
+### Error Handling
+
+All semantically valid requests accepted for processing MUST be returned with the HTTP status code `200 OK`, even if the packet is ultimately rejected.
+
+An endpoint MAY return standard HTTP errors, including but not limited to: a malformed or unauthenticated request, rate limiting, or an unresponsive upstream service. Connectors SHOULD relay an ILP Reject packet back to the original sender with an appropriate [Final or Temporary error code](./0027-interledger-protocol-4/0027-interledger-protocol-4#error-codes). Server errors (status codes 500-599) SHOULD be translated into ILP Reject packets with `T00: Temporary Error` codes.
+
+### Idempotence Behavior
```

Comment:
What happens if a `Fulfill` packet is submitted with an idempotency key that the node on the receiving end of that request doesn't have a record for?

Reply:
Added a note that a node should return a `400` in this case: https://github.com/interledger/rfcs/compare/060732f4f52f286c020c9ebf085b453e91e51951...f4fc0020d82b46b6fdfde537a8f251906d95120c#diff-ccfda9b7df50a9cfd8618bb9c08e360aR94-R99